### PR TITLE
chore(i18n): Update pl.json

### PR DESCRIPTION
**Pull request by zwiastunsw**
<< Describe the changes >>
1. minor correction
2. add new constants into pl.json

### CODE REVIEW

**Comment by straker** [comment]
Thanks for the translation update. We'll find someone to review the changes who can speak Polish, so it may take a bit to get a review.

**Comment by zwiastunsw** [comment]
Thank you for the information.  
Can't we just agree that I take care of this translation?  
Being reviewed by someone who speaks Polish is certainly useful, but it is not just the translation that is important, it is the localization - adapting it to the language used in Poland for accessibility studies.   
I am one of the authors of the official translation of WCAG into Polish and many other important materials (How to meet WCAG, WCAG-EM, american "Trusted tester" system, RGAA 3.0 and 4.0 and other)

**Comment by straker** [comment]
As a policy we need every PR reviewed by someone who can verify the changes, so we'll still need to bring someone else in.

**Comment by zwiastunsw** [comment]
Then how about finding someone in Poland to do the review?

**Comment by WilcoFiers** [comment]
@zwiastunsw You are absolutely right. I reached out to a native Polish speaker who I know has worked in accessibility for several years. He said he'd look today.

**Comment by zwiastunsw** [comment]
@WilcoFiers, thank you.
I am still working on this translation. In many places the language can still be simplified. But it will take some time.

File: locales/pl.json [hunk 1]
```diff
@@ -599,11 +630,11 @@
     },
     "alt-space-value": {
       "pass": "Element ma poprawną wartość atrybutu alt.",
-      "fail": "Element ma atrybut alt zawierający tylko znak spacji, który nie jest ignorowany przez wszystkie czytniki ekranu."
+      "fail": "Element ma atrybut alt zawierający tylko znak spacji, który nie przez wszystkie czytniki ekranu jest ignorowany ."
```
Comment (ajanec01) [inline locales/pl.json:633]: I'd preferr the original version but the new one is fine too.

File: locales/pl.json [hunk 2]
```diff
@@ -743,7 +774,7 @@
     "skip-link": {
       "pass": "Cel łącza pomijajacego istnieje.",
       "incomplete": "Cel łącza pomijania powinien być widoczny po aktywacji.",
-      "fail": "Nie ma celu łącza pomijającego."
+      "fail": "Nie istniej cel łącza pomijającego."
```
Comment (ajanec01) [inline locales/pl.json:777]:
```suggestion
      "fail": "Nie istnieje cel łącza pomijającego."
```

File: locales/pl.json [hunk 3]
```diff
@@ -872,7 +903,7 @@
       "fail": "W HTML 5, atrybuty scope mogą być używane tylko w elementach nagłówkowych tabeli (<th>)."
     },
     "same-caption-summary": {
-      "pass": "Treść atrybutu summary i <caption> nie są powielane.",
+      "pass": "Treści atrybutu summary i <caption> nie są powielane.",
```
Comment (ajanec01) [inline locales/pl.json:906]:
```suggestion
      "pass": "Treści atrybutu summary i elementu <caption> nie są powielane.",
```

**Review by ajanec01 — CHANGES_REQUESTED** [review]
Some minor editorial suggestions. Good job overall.

File: locales/pl.json [hunk 4]
```diff
@@ -39,11 +39,11 @@
     },
     "aria-meter-name": {
       "description": "Każdy węzeł meter oparty na ARIA ma dostępną nazwę.",
-      "help": "Węzły liczników ARIA muszą mieć dostępną nazwę."
+      "help": "Węzły metter ARIA muszą mieć dostępną nazwę."
```
[the remark this answers is not in the document]
Comment (zwiastunsw) [reply locales/pl.json:42]: :) thx. Noted, but I'm correcting some more. 
      "description": "Każdy licznik (meter) oparty na ARIA ma dostępną nazwę.",
      "help": "Liczniki (meter) ARIA muszą mieć dostępną nazwę."

File: locales/pl.json [hunk 5]
```diff
@@ -55,27 +55,31 @@
...
       "description": "Atrybut aria-roledescription jest używany tylko w elementach, które mają rolę określoną domyślnie lub jawnie.",
       "help": "Użyj aria-roledescription w elementach o roli semantycznej."
     },
     "aria-roles": {
-      "description": "Zapewnia, że wszystkie elementy z atrybutem roli używają prawidłowej wartości.",
+      "description": "Wszystkie elementy z atrybutem roli używają prawidłowej wartości.",
       "help": "Stosowane role ARIA muszą być zgodne z obowiązującymi wartościami."
     },
+    "aria-text": {
+      "description": "Upewnijsię, że \"role=text\" jest używany dla elementów, które nie mają potomków przyjmujących fokus",
```
[the remark this answers is not in the document]
Comment (zwiastunsw) [reply locales/pl.json:69]: "na elementach" nie jest poprawnym związkiem wyrazów w tym kontekście. 
Koryguję ten wiersza na:
      "description": "Atrybut \"role=text\" jest używany dla elementów, które nie mają potomków przyjmujących fokus",

File: locales/pl.json [hunk 6]
```diff
@@ -145,17 +149,25 @@
...
+    },
     "focus-order-semantics": {
-      "description": "Elementy w porządku fokusowania mają odpowiednią rolę.",
-      "help": "Elementy w porządku fokusowania muszą mieć rolę odpowiednią dla treści interaktywnych."
+      "description": "Elementy w porządku otrzymywania fokusu mają odpowiednią rolę.",
+      "help": "Elementy w porządku otrzymywania fokusu muszą mieć rolę odpowiednią dla treści interaktywnych."
     },
     "form-field-multiple-labels": {
-      "description": "Pole formularza nie posiada wielu  etykiet (elementów label).",
+      "description": "Pole formularza nie ma wielu  etykiet (elementów label).",
```
[the remark this answers is not in the document]
Comment (zwiastunsw) [reply locales/pl.json:161]: Pełna zgoda

File: locales/pl.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (ajanec01) [reply locales/pl.json:42]: Yep, sounds better.

File: locales/pl.json [hunk 7]
```diff
@@ -145,17 +149,25 @@
...
       "help": "Pole formularza nie powinno zawierać wielu elementów label."
     },
+    "frame-focusable-content": {
+      "description": "Upewnij się, że elementy <frame> i <iframe> z treścią przyjmującą fokus nie mają tabindex=-1",
+      "help": "Ramki z treścią przyjmującą fokus nie mogą mieć tabindex=-1"
+    },
     "frame-tested": {
-      "description": "Elementy <iframe> i <frame> zawierają skrypt axe-core.",
-      "help": "Ramki muszą być testowane ze skryptem axe-core."
+      "description": "Elementy <iframe> i <frame> muszą być testowane z axe-core.",
```
[the remark this answers is not in the document]
Comment (zwiastunsw) [reply locales/pl.json:169]: the new version reflects the intentions of the original, in the previous version the intentions were incomprehensible

File: locales/pl.json [hunk 8]
```diff
@@ -275,7 +287,7 @@
     },
     "meta-refresh": {
       "description": "<meta http-equiv=\"refresh\" nie jest używane.",
-      "help": "Automatyczne odświeżenie nie być stosowane."
+      "help": "Automatyczne odświeżenie strony nie być stosowane."
```
[the remark this answers is not in the document]
Comment (zwiastunsw) [reply locales/pl.json:290]: Thx. I correct it
      "description": "<meta http-equiv=\"refresh\"> nie jest stosowane.",
      "help": "Automatyczne odświeżenie strony **nie może być** stosowane."

File: locales/pl.json (hunk 3 above)
Comment (zwiastunsw) [reply locales/pl.json:906]: @ajanec01 : Thank you for your valuable comments. I will add the revised version again tomorrow. I have made a few other minor changes to it. Can I ask you to review it again tomorrow?

File: locales/pl.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (ajanec01) [reply locales/pl.json:42]: Also, I wasn't sure about which English word "Węzeł" was referring to, is it `range` (the super class). Seemed unusual to me but I don't use this language on a day by day basis.

File: locales/pl.json (hunk 3 above)
Comment (ajanec01) [reply locales/pl.json:906]: Sure!

**Review by ajanec01 — APPROVED** [review]
Minor typo, otherwise approving.

**Comment by WilcoFiers** [comment]
@StevenDufresne wrote:
> I am still working on this translation. In many places the language can still be simplified. But it will take some time.

I'm assuming this is done, correct? Can you confirm. We're doing a release today, so if it's finalised we'll be able to included it right away.

**Comment by zwiastunsw** [comment]
@WilcoFiers Yes, it's ready  :)
thx@ajanec01 :)

**Comment by ajanec01** [comment]
Pleasure!

**Review by WilcoFiers — APPROVED** [review]
Reviewed for security.

**Pull request merged**